docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,95 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+Signals are similar to ops, but are not stored; they're useful to implement features like user presence. They provide a simple way to communicate transient data to connected clients within a Fluid application.
```

Comment:
Our documentation teaches customers about ops, but it's not really something they use directly (instead they'll be indirectly generating ops by interacting with a DDS).  Maybe prefer to contrast against DDS usage rather than ops directly.